<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveEditorWindow</class>
 <widget class="QMainWindow" name="SaveEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaveEditorWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1071</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="networkTab">
     <attribute name="title">
      <string>Network</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>1051</width>
        <height>771</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTreeWidget" name="networkTree">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="indentation">
          <number>8</number>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>125</number>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>IP</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>323</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>321</width>
            <height>151</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="computerConfig">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">margin-top: 4px;</string>
             </property>
             <property name="text">
              <string>Computer Name:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>IP Address:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Special:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Internal ID:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Type:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="computerNameEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px; margin-top: 4px;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="computerIPEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="computerIDEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="computerTypeEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
             <item>
              <property name="text">
               <string>Corporate</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Home</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Server</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Empty</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>eOS Device</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="computerSpecEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Player's Mail Server</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Player's Computer</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="formLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>200</y>
            <width>321</width>
            <height>101</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="netmapConfig">
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">margin-top: 4px;</string>
             </property>
             <property name="text">
              <string>X Position:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="netmapXEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px; margin-top: 4px;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Y Position:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="netmapYEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="netmapVisibleEdit">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">margin-left: 5;</string>
             </property>
             <property name="text">
              <string>Known?</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>323</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background:lightgrey</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>180</y>
            <width>323</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background:lightgrey</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Netmap&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>310</y>
            <width>323</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background:lightgrey</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Security&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QWidget" name="formLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>330</y>
            <width>321</width>
            <height>161</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="securityConfig">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="styleSheet">
              <string notr="true">margin-top: 4px;</string>
             </property>
             <property name="text">
              <string>Level:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="securityLevelEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px; margin-top: 4px;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Trace Time:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="securityTraceTimeEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Ports to Crack:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="securityPortsToCrackEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Admin IP:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="securityAdminIPEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Proxy Time:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="securityProxyTimeEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>500</y>
            <width>323</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background:lightgrey</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Admin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QWidget" name="formLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>520</y>
            <width>321</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="adminConfig">
           <item row="0" column="0">
            <widget class="QLabel" name="label_16">
             <property name="styleSheet">
              <string notr="true">margin-top: 4px;</string>
             </property>
             <property name="text">
              <string>Type:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="adminTypeEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px; margin-top: 4px;</string>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Basic</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fast</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="adminResetPassEdit">
             <property name="styleSheet">
              <string notr="true">margin-left: 5;</string>
             </property>
             <property name="text">
              <string>Password Resets?</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="adminSuperEdit">
             <property name="styleSheet">
              <string notr="true">margin-left: 5;</string>
             </property>
             <property name="text">
              <string>Super?</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>620</y>
            <width>323</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background:lightgrey</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Firewall&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QWidget" name="formLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>640</y>
            <width>321</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="firewallConfig">
           <item row="0" column="0">
            <widget class="QLabel" name="label_18">
             <property name="styleSheet">
              <string notr="true">margin-top: 4px;</string>
             </property>
             <property name="text">
              <string>Complexity:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="firewallComplexityEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px; margin-top: 4px;</string>
             </property>
             <property name="minimum">
              <number>-5</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Solution:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="firewallSolutionEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Additional Delay:</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="firewallDelayEdit">
             <property name="styleSheet">
              <string notr="true">margin-right: 5px;</string>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="metaTab">
     <attribute name="title">
      <string>Metadata</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenSave"/>
    <addaction name="actionWriteSave"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenSave">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionWriteSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
